### PR TITLE
New package: gajim-plugin_installer-1.3.8

### PR DESCRIPTION
**Pull request by teldra**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me

#### Does it build and run successfully? 
(Please choose at least one native build and, if supported, at least one cross build. More are better.)
- [x] I built this PR locally for my native architecture, (x86_64-glibc)
- [ ] I built this PR locally for these architectures (if supported. mark crossbuilds):
  - [ ] aarch64-musl
  - [ ] armv7l
  - [ ] armv6l-musl

Please merge #28980 first.

### CODE REVIEW

File: srcpkgs/gajim-plugin_installer/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'gajim-plugin_installer'
+pkgname=gajim-plugin_installer
+_gajim_version=1.3
+version=1.3.8
```
Comment (ericonr) [inline srcpkgs/gajim-plugin_installer/template:4]:
```suggestion
version=1.3.8
_gajim_version=${version%.*}
```

That said, where does `version` come from, given that we are using only `_gajim_version` to download the distfile?

Comment (teldra) [reply srcpkgs/gajim-plugin_installer/template]: It looks like, upstream uses this naming scheme. So I see no reason to not implement `${version%.*}` in the url. Imho everything is clear with this.

**Comment by ericonr** [comment]
My point is, what does the `1.3.8` mean if we download from `1.3`? When it's updated, what has changed?

**Comment by teldra** [comment]
> My point is, what does the `1.3.8` mean if we download from `1.3`? When it's updated 1.3.9, what has changed?

Ah, my bad. The `_gajim_version` is the major gajim release. The plugins are sorted in versioned folders. So it even makes no sense what I stated before. The _gajim variable has nothing to do with the version of this particular plugin. Increase it, when gajim version changes. Hopefully with a consistent naming scheme, it wil create less maintining overhead in the future.
The situation is a bit weird with this plugin: When I get this correct, up to now, the plugin downloaded from the master dir on their ftp. Now the folder is versioned. The older plugin (installations of gajim pre 1.3) is incompatible with new version of gajim. The old plugin will load from master, where plugins for the gajim devel version are stored. In the future, I will probably fix gajim-omemo also.

File: srcpkgs/gajim-plugin_installer/template (hunk 1 above)
Comment (teldra) [reply srcpkgs/gajim-plugin_installer/template]: https://github.com/void-linux/void-packages/pull/28982#issuecomment-794646320

**Comment by ericonr** [comment]
I think it would be more correct to version it as `1.3`, then, even if it can be a tad more confusing. Leave a comment if you think it's necessary, maybe.

**Comment by teldra** [comment]
There are different and probably better ways to install this plugin because of upstream versioning. (https://dev.gajim.org/gajim/gajim-plugins/-/wikis/home)

**Pull request closed without merging**